arches/install/arches-templates/project_name/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}{% autoescape off %}
+
+{% trans "Dear" %} {{ user.get_short_name }}{% trans "," %}
```

Comment:
This should use a block trans instead of 2 individual trans statements

Reply:
Amended as per suggestion